sqtom/fitting_1d.py

```diff
@@ -25,23 +25,90 @@
 """
 
 
+import warnings
 import numpy as np
+from scipy.optimize import root_scalar
 from lmfit import Minimizer, Parameters
 from sqtom.forward_solver import degenerate_pmf
 
 
+def two_schmidt_mode_guess(pd_data, sq_label="sq_", noise_fraction=0.001, atol=1e-8):
+    """Given a single mode histogram, this function generates a "physically" motivated guess for the loss, Schmidt
+    occupations and dark counts parameters.
+
+    This model is sensible only if there is a probability for an n == 2 event. If there are only probabilities for
+    n == 0 and n == 1 then the g2 will erroneously be calculated as 0 and as a result the remainder of the estimtes will
+    be nonsense.
+
+    Args:
+        pd_data (array): rectangular array with the probability mass functions of the photon events
+        sq_label (string): label for the squeezing parameters.
+
+    Returns:
+        dict: dictionary containing a set of "reasonable" model parameters. In the case that the pd_data only has
+        entries for n == 0, or n == 0 and n == 1, it returns None along with a warning message.
```

Comment:
This is just a suggestion for making sure that sphinx will render code like parts of the text well.

@nquesada would sphinx documentation be intended to be included for the repo down the line? (If not, then not applicable.)
```suggestion
        dict: dictionary containing a set of "reasonable" model parameters. In the case that the ``pd_data`` only has
        entries for ``n == 0``, or ``n == 0`` and ``n == 1``, it returns ``None`` along with a warning message.
```

Reply:
Same here.